verilog/formatting/tree_unwrapper.cc

```diff
@@ -896,6 +921,22 @@ void TreeUnwrapper::Visit(const verible::SyntaxTreeLeaf& leaf) {
   // up to a newline.
   LookAheadBeyondCurrentLeaf();
 
+  const std::regex uvm_end("^`uvm_.*_end$", std::regex_constants::basic);
+  if ((static_cast<int>(tag) == yytokentype::MacroIdItem) &&
+      (std::regex_search(std::string(leaf.get().text), uvm_end))) {
+    for (auto* itr = CurrentTokenPartition()->PreviousSibling() ; itr ;
```

Comment:
`const auto*`

Reply:
I can't add `const` keyword because I'm using `ApplyPreOrder` with `SetIndentationSpaces` modifier:
https://github.com/google/verible/pull/113/commits/0dc48bc8028e73bbc980bf25f7564216a13ee162#diff-7872d97b05fbc6f2a64cf14b8ae74d75R957